Assets/Scripts/Models/Buildable/Utility.cs

```diff
@@ -549,6 +551,15 @@ public void Deconstruct(Utility utility)
             Removed(this);
         }
 
+        if (deconstructInventory != null)
+        {
+            foreach (Inventory inv in deconstructInventory)
+            {
+                inv.MaxStackSize = PrototypeManager.Inventory.Get(inv.Type).maxStackSize;
```

Comment:
This is actually needed. Inventory max sizes aren't available when the Furniture.xml file is read in. This location seemed to be the easiest to squeeze in the max size.